.idea/codeStyleSettings.xml

```diff
@@ -289,6 +289,9 @@
         <JavaCodeStyleSettings>
           <option name="DO_NOT_WRAP_AFTER_SINGLE_ANNOTATION" value="true" />
         </JavaCodeStyleSettings>
+        <MarkdownNavigatorCodeStyleSettings>
+          <option name="RIGHT_MARGIN" value="72" />
```

Comment:
Not sure, I installed the markdown plugin for android studio, I guess it added this. I can remove it if needed.